tests/python/unittest/test_gluon.py

```diff
@@ -51,6 +51,34 @@ def test_paramdict():
     params.load('test.params', mx.cpu())
 
 
+@with_seed()
+def test_constant():
+    class Test(gluon.HybridBlock):
+        def __init__(self, **kwargs):
+            super(Test, self).__init__(**kwargs)
+            self.value = np.asarray([[1,2], [3,4]])
+            self.const = self.params.get_constant('const', self.value)
+
+        def hybrid_forward(self, F, x, const):
+            return x + const
```

Comment:
Should it be `x + self.const`?

Reply:
const is the parameter, and block handles it.